src/Client.php

```diff
@@ -277,15 +277,15 @@ public function updateDocument($id, $data, $index, array $options = []): Respons
 
             $docOptions = $data->getOptions(
                 [
-                    'version',
-                    'version_type',
-                    'routing',
-                    'percolate',
-                    'parent',
-                    'retry_on_conflict',
                     'consistency',
-                    'replication',
+                    'if_primary_term',
+                    'if_seq_no',
```

Comment:
@deguif @ruflin after reading the docs, I think we should remove those from the default params. It shouldn't be forced since there are set when it gets a document right now